.vale/styles/Localization/PhrasalVerbs.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
General comment for this rule - as it is laid out below, there will be a huge amount of false positives for this rule. Need to reduce the false positives to make this useful.